CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 #### Improvements
 
+- feat: enable use of replica database (delegating the choice to `DATABASES_ROUTER`) ([#359](https://github.com/jazzband/django-auditlog/pull/359))
```

Comment:
Please place in a new section above because this section belongs to the already released version 1.0.0

Reply:
Also, please mention `LogEntry no longer save to same database instance is using`